Testing Support/FormattedExample.m

```diff
@@ -212,20 +215,23 @@ BOOL CStyleMethod()
 
 INSAFSuccessBlock INSAPIClientModelSuccessHandler(Class mantleClass, NSString *__nullable keyPath, INSHTTPSuccess __nullable success, INSHTTPFailure __nullable failure)
 {
-    return INSAPIClientModelSuccessChain(mantleClass, keyPath, ^(__kindof INSModel *model, id _) {
-        if (success) {
-            success(model);
-        }
-    }, failure);
+    return INSAPIClientModelSuccessChain(
```

Comment:
This is the only one change that I think might be somewhat contentious. It might be worth shooting an email to seller-ios@ about this behavior change, but I don't have concerns with it personally.

Reply:
this is interesting. I'd expect either putting new lines between each, or none, but this change does a mixture. new line, with all args on the same line, then same line on "end" of the block. If I had the choice I'd forgo this change but not really enough of a concern to block